docs/sphinx/using/backends/simulators.rst

```diff
@@ -169,6 +169,7 @@ To execute a program on the multi-node multi-GPU NVIDIA target, use the followin
     If a target is set in the application code, this target will override the :code:`--target` command line flag given during program invocation.
 
     .. note::
+        
```

Comment:
It is still the same... See: https://htmlpreview.github.io/?https%3A%2F%2Fraw.githubusercontent.com%2FNVIDIA%2Fcuda-quantum%2Fdocs%2Fpreview%2Fpr-2208%2Fusing%2Fbackends%2Fsimulators.html=

Shouldn't they be number serially, i.e., (1) and (2) or perhaps use bullets?

Reply:
Yes the original version was a hardcoding of the serial numbering but markdown didn't render that correctly, so I tried the markdown ordered numbering syntax and that didn't work either.  Your suggestion to just change it to bullet points is excellent. I'll add that commit.